packages/components/src/drop-zone/index.tsx

```diff
@@ -25,6 +25,71 @@ import {
 import type { DropType, DropZoneProps } from './types';
 import type { WordPressComponentProps } from '../context';
 
+function DropIndicator( { label }: { label?: string } ) {
+	const disableMotion = useReducedMotion();
+	const backdrop = {
```

Comment:
Nit: Given we continue to work with framer motion for this animation (which @DaniGuardiola is looking at changing in https://github.com/WordPress/gutenberg/issues/60975), those animation variants could be declared outside of the component, there's no need to redefine them on every `DropIndicator` render.